### PR TITLE
Added kubevirt bugs to RN

### PR DESCRIPTION
**Pull request by ctauchen**
* Added a collection of 4.8 bugs for the Console Kubevirt Plugin
* [direct preview -- scroll down to Console Kubevirt Plugin](https://deploy-preview-34207--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-bug-fixes)

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -1307,7 +1307,23 @@ As part of the continued deprecation of the Operator package manifest format, th
 
 *Console Kubevirt Plugin*
 
+* Previously, in a web console form that helped users add boot sources to virtualization templates, an explanatory text gave information only for Fedora, regardless of what operating system the template used. This update adds a fix that provides examples that are specific to the template's operating system so users have relevant guidance. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1908169[*BZ#1908169*])
 
+* Previously, in a web console wizard that helped users create virtual machine templates, imprecise language made it unclear whether an operation applied to the template or to a virtual machine. This fix clarifies the description so users can make an informed decision. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1922063[*BZ#1922063*])
+
+* In the web console, a vague error message caused unnecessary confusion for some users who tried to add a network interface to a virtual machine they were creating from a template. This update adds detail to the error message so users can troubleshoot the error more easily. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1924788[*BZ#1924788*])
```
Comment (mburke5678) [inline release_notes/ocp-4-8-release-notes.adoc:1314]: Do you want a _Previously_ here to be consistent?

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -1307,7 +1307,23 @@ As part of the continued deprecation of the Operator package manifest format, th
...
+
+* Previously, a drop-down list in the web console was obscured by button elements, leaving users unable to select certain operating systems when creating a virtual machine. This fix includes an adjustment to the button elements' `z-index` value, which corrects the error and lets users select any of the available operating systems. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1930015[*BZ#1930015*])
+
+* Previously, if you used the web console's new virtual machine wizard on a cluster with no defined storage classes, the web console got stuck in an infinite loop and crashed. This fix removes the storage class drop-down list in instances where no storage classes are defined. As a result, the web console does not crash. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1930064[*BZ#1930064*]) 
+
+* Previously, text in a button element didn't clearly describe the button's function, which was to remove a VM template from a list of favorites. This fix updates the text to clarify what the button does. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1937941[*BZ#1937941*])
```
Comment (mburke5678) [inline release_notes/ocp-4-8-release-notes.adoc:1322]: s/didn't/did not
s/which was/which is

**Comment by mburke5678** [comment]
@ctauchen A few nits. Otherwise LGTM

File: release_notes/ocp-4-8-release-notes.adoc (hunk 2 above)
Comment (ctauchen) [reply release_notes/ocp-4-8-release-notes.adoc:1408]: Thanks -- all changed as suggested.

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (ctauchen) [reply release_notes/ocp-4-8-release-notes.adoc:1400]: Thanks -- all changed as suggested.

**Review by jeana-redhat — APPROVED** [review]

**Pull request merged**